com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs

```diff
@@ -533,7 +533,11 @@ void IViewDisplayer.SetTexture(ViewCompositionIndex index, Texture texture)
             }
         }
 
-        void IViewDisplayer.Repaint() => Repaint();
+        void IViewDisplayer.Repaint()
+        {
+            RefreshLibraryDisplay();
```

Comment:
Repaint already call this. It seams not needed.

Reply:
Done!